pywinauto/application.py

```diff
@@ -1247,6 +1248,32 @@ def kill(self):
     # Non PEP-8 aliases
     kill_ = Kill_ = kill
 
+    def is_process_running(self):
+        """
+        Checks that process is still running
+
+        Returns True if process is still running otherwise - False
+        """
+        is_running = False
+        try:
+            h_process = win32api.OpenProcess(
+                win32con.PROCESS_QUERY_INFORMATION,
+                0,
+                self.process)
+            is_running = win32process.GetExitCodeProcess(
+                h_process) == win32defines.PROCESS_STILL_ACTIVE

```

Comment:
What will happen if ```win32api.OpenProcess``` doesn't raise the exception, but h_process is still NULL (check [MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684320%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)) ? Will ```win32process.GetExitCodeProcess``` raise the same exception ? I'd add a unit test for it. Leverage mock facilities to return 0 from ```OpenProcess```